views/index.pug

```diff
@@ -61,7 +61,7 @@ html
                 div.logo
                   if logo
                     img(src=`data:image/png;base64,${logo.toString('base64')}`, height="24px", alt="Logo")
-            td= message.to
+            td= `${message.to} BCC: ${message.bcc}`
```

Comment:
I think this needs to be improved, i.e if there is no BCC, it should not show up here. 
Also would be nice to have CC column as an addition

Reply:
Can we do bcc and cc as separate rows?

Reply:
Done as separate rows!